business/istio_config.go

```diff
@@ -1020,11 +1020,14 @@ func (in *IstioConfigService) IsAmbientEnabled() bool {
 	}
 	if ambientEnabled == nil || currentTime.Sub(*lastUpdateTime) > time.Minute {
 		ambientEnabled = new(bool)
-		daemonset, err := in.kialiCache.GetDaemonSet(in.config.IstioNamespace, "ztunnel")
+		selector := map[string]string{
+			"app": "ztunnel",
+		}
+		daemonset, err := in.kialiCache.GetDaemonSetsWithSelector(meta_v1.NamespaceAll, selector)
 		if err != nil {
-			log.Debugf("No ztunnel found in istio namespace: %s ", err.Error())
+			log.Debugf("No ztunnel found in Istio Ambient namespace (%s): %s ", in.config.ExternalServices.Istio.IstioAmbientNamespace, err.Error())
```

Comment:
This log statement isn't accurate anymore since kiali is checking all namespaces.

Reply:
Changed the message